yarn.lock

```diff
@@ -1972,6 +1972,14 @@
     get-stream "^6.0.1"
     minimist "^1.2.5"
 
+"@mapbox/geojson-rewind@^0.5.1":
```

Comment:
wow mapbox sure pulls in a lot of dependencies.

Reply:
Yes, I didn't know about a lot of them